src/adb.js

```diff
@@ -332,6 +333,29 @@ class Adb {
     });
   }
 
+  // Get OS name
+  getOsName() {
+    var _this = this;
+    return new Promise(function(resolve, reject) {
+      _this
+        .shell(["cat", "/etc/system-image/channel.ini", "|grep tag="])
```

Comment:
What information do you want here, exactly? The tag is usually in the same line with the rest of the `version-detail`.

Maybe we could rename this function to avoid confusion with `getOs`. Maybe `getUbuntuTouchVersionDetail`.

It would also be better to parse the response in javascript, rather than piping through grep.